pkg/manager/manager.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
```

Comment:
This could be a good fit for the existing `controller` package, which would avoid "stuttering" for importers - they'd use `controller.NewManager` (which reads more like "new controller manager") rather than `manager.NewManager`.

Reply:
That's a really good point, will move.